land_use/serializers.py

```diff
@@ -0,0 +1,88 @@
+from decimal import Clamped
+from pyexpat import model
+from django.db import models
+from rest_framework import serializers
+from rest_framework_gis import serializers as gis_serializers
+
+from land_use import models
+
+
+class LandUseSerializer(gis_serializers.GeoFeatureModelSerializer):
+    """Serializer for geographic `models.LandUseClasses` spatial data."""
+    class Meta:
+        """Meta calss for geographic data `LandUseSerializer` serializer."""
+        model = models.LandUseClasses
+        id_field = False
+        geo_field = 'geom'
+        fields = (
+            'id',
+            'geom',
+        )
+
+
+class LandUseDetailSerializer(serializers.ModelSerializer):
+    """Serializer for detailed `models.LandUseClasses` data."""
+    class Meta:
+        """Meta class for `LandUseDetailSerializer` serializer."""
+        model = models.LandUseClasses
+        id_field = False
+        fields = (
+            'id',
+            'sg_uf',
+            'no_ti',
+            'co_funai',
+            'dt_homologada',
+            'ds_cr',
+            'co_cr',
+            'nu_ano',
+            'no_estagio',
+            'no_satelites',
+            'nu_resolucoes',
+            'dt_imagens',
+            'nu_area_km2',
+            'nu_area_ha',
+            'dt_cadastro',
+        )
+
+
+class LandUseYearsSerializer(serializers.ModelSerializer):
```

Comment:
This filter is not for each Funai Indigenous Lands?

Reply:
The operation is correct. For better understanding, docString was added in views.py/LandUserYearsVeiw.